docs/TESTING.md

```diff
@@ -0,0 +1,44 @@
+## Testing
+
+Tests are performed using real Gradescope accounts to interact with the Gradescope server. As such, valid Gradescope accounts and courses are required to run the tests.
+
+### Environment
+
+Create an `.env` file in the root directory of the project with the following environment variables:
```

Comment:
Glad that you added this. Because adding in Bash file doesnot work in this scenario